src/js/map.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable object-shorthand */
+/* eslint-disable prefer-template */
+/* eslint-disable func-names */
+/* eslint-disable no-undef */
+mapboxgl.accessToken = 'pk.eyJ1IjoidmVyYXNpYXZrYSIsImEiOiJja3JkaXk1N3k1Y2k5Mm9xcDNiNnJhZ3czIn0.VVSKa_qR-1yqHrKVi7XE0w';
```

Comment:
[Note for the future and real-life project]
Access tokens should not be stored in the codebase itself. Better place for them are Environment variables.

In this task it is okay: )